src/sentry/utils/event.py

```diff
@@ -123,5 +123,4 @@ def track_event_since_received(
             "reprocessing": "true" if reprocessing2.is_reprocessed_event(event_data) else "false",
             "type": event_data.get("type"),
         },
```

Comment:
@JoshFerge moving the discussion from the other pr here - i see `sample_rate: float = settings.SENTRY_METRICS_SAMPLE_RATE` in `metrics.py` which gets passed down which is why i thought it was actually impacting the sample rate